content/article/announcing-dancer2-2-0-0.md

```diff
@@ -0,0 +1,99 @@
++++
+canonicalUrl=""
+categories="web"
+date=2025-09-12T18:12:41
+description="The Dancer Core Team is excited to announce the release of Dancer2 2.0.0."
+draft=false
+image=""
+thumbnail=""
```

Comment:
I just noticed at https://www-perl-com-pr-480.onrender.com/article/announcing-dancer2-2-0-0/ that there is no preview image. Can you add a preview image and a thumbnail? They make a huge difference when the article gets shared around, as articles without images are much less eye catching. You can see that without the thumbnail, the default image is your photo: https://www-perl-com-pr-480.onrender.com/ If you're happy with that, we'd just need to add the `image` for the article as that's what gets previewed when a link is shared.